python-package/lightgbm/basic.py

```diff
@@ -2816,10 +2832,14 @@ def predict(self, data, num_iteration=None,
         data : string, numpy array, pandas DataFrame, H2O DataTable's Frame or scipy.sparse
             Data source for prediction.
             If string, it represents the path to txt file.
+        start_iteration : int, optional (default=0)
+            Start index of the iteration to predict.
+            If None or <= 0, starts from the first iteration.
         num_iteration : int or None, optional (default=None)
             Limit number of iterations in the prediction.
-            If None, if the best iteration exists, it is used; otherwise, all iterations are used.
-            If <= 0, all iterations are used (no limits).
+            If None, if the best iteration exists and start_iteration is None or <= 0, it is used;
+            otherwise, all iterations from start_iteration are used.
+            If <= 0, all iterations from start_iteration are used (no limits).
```

Comment:
Now it is unclear what `it` means.
```suggestion
            If None, if the best iteration exists and ``start_iteration`` is None or <= 0, the best iteration is used;
            otherwise, all iterations from ``start_iteration`` are used.
            If <= 0, all iterations from ``start_iteration`` are used (no limits).
```